### ISSUE TITLE
TauAnalysis/MCEmbeddingTools unit test fails

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#29897]
The unit test in `TauAnalysis/MCEmbeddingTools` fails since CMSSW_11_1_X_2020-05-18-1100
```
----- Begin Fatal Exception 18-May-2020 13:37:51 CEST-----------------------
An exception of category 'ScheduleExecutionFailure' occurred while
   [0] Calling beginJob
Exception Message:
Unrunnable schedule
Module run order problem found: 
rawDataCollector consumes gmtStage2Raw, gmtStage2Raw consumes simEmtfDigis, simEmtfDigis consumes rpcRecHits, rpcRecHits consumes muonRPCDigis, muonRPCDigis consumes rawDataCollector
 Running in the threaded framework would lead to indeterminate results.
 Please change order of modules in mentioned Path(s) to avoid inconsistent module ordering.
----- End Fatal Exception -------------------------------------------------
```
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc820/CMSSW_11_1_X_2020-05-18-1100/unitTestLogs/TauAnalysis/MCEmbeddingTools#/

**Comment by makortel** [comment]
assign simulation, analysis

**Comment by makortel** [comment]
Appears to be caused by https://github.com/cms-sw/cmssw/pull/29767 that made `simEmtfDigis` to consume `rpcRecHits`.

**Comment by makortel** [comment]
assign l1

**Comment by makortel** [comment]
> Appears to be caused by #29767 that made `simEmtfDigis` to consume `rpcRecHits`.

By quick look that dependence could be removed rather easily https://github.com/cms-sw/cmssw/pull/29767/files#r426686938.

**Comment by makortel** [comment]
Fixed in #29899

**Comment by santocch** [comment]
+1
